src/redux/types/types.ts

```diff
@@ -3,3 +3,31 @@ export const forgetPasswordRequest = 'forgetPasswordRequest';
 export const forgetPasswordSuccess = 'forgetPasswordSuccess'
 
 export const forgetPasswordFailed = 'forgetPasswordFailed'
```

Comment:
Why is this file called types when there's no types definitions here? `types.ts` in js world implies it's the file with all typedefs used by the module. We should name our files more concrete. In this case it might be `actionTypes.ts`. Also please pay attention to naming of constants and redux actions